components/Layout.tsx

```diff
@@ -35,32 +35,32 @@ function Head(): ReactElement {
   return (
     <HTMLHead>
       {/* Primary Meta Tags */}
-      <title>Loot</title>
-      <meta name="title" content="Loot" />
+      <title>Adventure Cards</title>
+      <meta name="title" content="Adventure Cards" />
       <meta
         name="description"
-        content="Loot is randomized adventurer gear generated and stored on chain."
+        content="Adventure Cards is a collection of randomized trading cards generated and stored on chain."
       />
 
       {/* OG + Faceook */}
       <meta property="og:type" content="website" />
-      <meta property="og:url" content="https://www.lootproject.com/" />
-      <meta property="og:title" content="Loot" />
+      <meta property="og:url" content="https://0xadventures.com/" />
+      <meta property="og:title" content="Adventure Cards" />
       <meta
         property="og:description"
-        content="Loot is randomized adventurer gear generated and stored on chain."
+        content="Adventure Cards is a collection of randomized trading cards generated and stored on chain."
       />
-      <meta property="og:image" content="https://lootproject.com/meta.png" />
+      <meta property="og:image" content="https://0xadventures.com/meta.png" />
```

Comment:
looks like this is a 404. not sure if the image we've been seeing (twitter/discord pfps) is available anywhere on 0xadventures.com yet.